src/jenkins_job_insight/analyzer.py

```
@@ -1669,12 +1676,16 @@ async def analyze_job(
                         repo_path / repo_name,
                         depth=50,
                         branch=tests_ref,
+                        token=tests_repo_token or None,
                     )
                     cloned_repos[repo_name] = repo_path / repo_name
                     repo_context = f"\nTest repository cloned from: {clean_tests_url} (at {repo_name}/)"
                 except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)